lib/OpenQA/Utils.pm

```diff
@@ -606,20 +553,10 @@ sub create_git_clone_list ($job_settings, $clones = {}) {
     return $clones;
 }
 
-sub _round_a_bit {
-    my ($size) = @_;
-
-    if ($size < 10) {
-        # give it one digit
-        return int($size * 10 + .5) / 10.;
-    }
-
-    return int($size + .5);
-}
-
-sub human_readable_size {
-    my ($size) = @_;
+# give it one digit
```

Comment:
Is this comment really necessary?

Reply:
Please be aware that I did not introduce it. I just moved the comment according to the integration of signatures. I recommend to not change this as part of one commit as that is unrelated to introducing signatures